cyder/templates/base/tables.html

```diff
@@ -45,3 +45,48 @@
 
 {% endif %}
 {% endmacro %}
+
+{# render a table containing a single object so that it doesn't look like a table #}
+{% macro render_object(request, object_table, class) %}
+{% if object_table %}
+  <table id="obj-info">
+
+    {% for row in object_table['data']%}
+      {% for col in row %}
+        {% set header, sort_field = object_table['headers'][loop.index0] %}
+         
+        <tr>
+          <td class="{{ header | lower }}_column"><b>{{ header }}</b></td>
+          
+          {% if col['value'] %}
+            <td>
+            {% for cell in col['value'] %}
+              {% if col['url'][loop.index0] %}
+                <a class="info" 
```

Comment:
"<a class="info"" shouldn't be there, unless it is specifically for the info column.